modern/src/reports/ReportLayoutPage.js

```diff
@@ -29,6 +35,18 @@ const ReportLayoutPage = ({ reportFilterForm:ReportFilterForm, setItems, childre
             <Paper className={classes.form}>
               <ReportFilterForm setItems={ setItems } />
             </Paper>
+            {showChartType && (
+              <Paper className={classes.chart}>
+                <FormControl variant="filled" margin="normal" fullWidth>
+                <InputLabel>{t('reportChartType')}</InputLabel>
+                <Select defaultValue="speed" onChange={e => setType(e.target.value)}>
```

Comment:
Looks like we define this default value in state and here. Can we avoid that?

Reply:
done